### PR TITLE
Dockerize featureService

### PR DESCRIPTION
**Pull request by c-w**
As part of the Fortis security review, we've established that the featureService should be deployed inside of the Fortis kubernetes cluster so that we don't have a single point of failure. More details on this discussion can be found in https://github.com/CatalystCode/project-fortis-pipeline/issues/222#issuecomment-365992791

In order to enable the featureService to be deployable to kubernetes we must dockerize it. As such, this pull request implements a Dockerfile and entry-point for the featureService that sets up the Node service, sets up the database schema, populates the database from a dump, etc. When starting the service via `docker run`, we now get a fully-functional deployment.

Additionally, this pull request also implements CD via Travis for the featureService to make it easy for us to publish new Docker images by simply authoring a release on Github. To facilitate the CD, the pull request also implements CI via Travis. Currently the CI just runs eslint so the pull request also includes assorted lint fixes.

I realize that the pull request is quite long, so here are pointers to the core of the new functionality:
- [docker-entrypoint.sh](https://github.com/CatalystCode/featureService/compare/dockerize?expand=1#diff-60713fb19f6c50a2473835df63ff17f9)
- [Dockerfile](https://github.com/CatalystCode/featureService/compare/dockerize?expand=1#diff-3254677a7917c6c01f55212f86c57fbf)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ADD config.js /app/config.js
+ADD controllers /app/controllers
+ADD services /app/services
+ADD ddl /app/ddl
+ADD docker-entrypoint.sh /app/docker-entrypoint.sh
+
+CMD /app/docker-entrypoint.sh
+
+ENV PORT=80
+EXPOSE 80
```
Comment (jcjimenez) [inline Dockerfile]: Do you know if it's possible to do a `EXPOSE $PORT`?

**Review by jcjimenez — APPROVED** [review]
LGTM w/minor question.

File: Dockerfile (hunk 1 above)
Comment (c-w) [reply Dockerfile]: Not sure. From the [docs/spec](https://docs.docker.com/engine/reference/builder/#expose) it seems like only constants can be exposed. I'll give it a try though and report back.

Comment (c-w) [reply Dockerfile]: Yes, seems to work. Good catch. See 453cccc

**Pull request merged**